Floofbot/Modules/Help.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Discord;
+using Discord.Addons.Interactive;
+using Discord.Commands;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Floofbot.Modules
+{
+    public class Help : InteractiveBase
```

Comment:
The help module and command can also get summaries

Reply:
This will be addressed in a future PR